vue.config.js

```diff
@@ -1,4 +1,22 @@
-const { defineConfig } = require('@vue/cli-service')
+const { defineConfig } = require('@vue/cli-service');
+const path = require('path');
+
 module.exports = defineConfig({
-  transpileDependencies: true
+  transpileDependencies: true,
+  css: {
+    loaderOptions: {
+      sass: {
+        additionalData: '@import "@/styles/styles.scss";',
+        implementation: require('node-sass')
+      },
+    },
+  },
+  configureWebpack: {
+    resolve: {
+      alias: {
+        '@': path.resolve(__dirname, 'src'),
+        '@components': path.resolve(__dirname, 'src/components'),
```

Comment:
remove this one and use the above one only. Then you will import like this -> @/components